src/main/java/faang/school/projectservice/controller/InternshipController.java

```diff
@@ -0,0 +1,45 @@
+package faang.school.projectservice.controller;
+
+import faang.school.projectservice.controller.validation.InternshipValidator;
+import faang.school.projectservice.dto.internship.InternshipDto;
+import faang.school.projectservice.dto.internship.InternshipFilterDto;
+import faang.school.projectservice.service.InternshipService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/projects")
+public class InternshipController {
+    private final InternshipValidator internshipValidator;
+    private final InternshipService internshipService;
+
+    @PostMapping("/create")
```

Comment:
for standart CRUD methods in springboot no need to write path name. Usually namings are given only for your own specific methods.  Just  @PostMapping and @PutMapping, @DeleteMapping. F.e. Not  @PostMapping("/create") , but  @PostMapping